### DISCUSSION TITLE
How is the graph generated?

### DISCUSSION THREAD

**Discussion by Jona898** [nrwl/nx#12760] [General]
I want to visualize for a home control system the dependencies which module sends data to another module.

The Graph visualization of nx (https://nx.dev/core-features/explore-graph) seems really good.
The Repository is really large. Can anyone point me to the files which generate the graph?

Thanks in advance
Jonathan

**Comment by zanettin** [comment]
i am also not 100% sure but my guess is https://github.com/nrwl/nx/tree/master/packages/nx/src/project-graph
